operator/v1/0000_70_dns-operator_00.crd.yaml-patch

```diff
@@ -0,0 +1,18 @@
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/upstreamresolvers/properties/upstreams/items/properties/address/anyOf
+  value:
+    - format: ipv4
+    - format: ipv6
+- op: add
+  path: /spec/versions/name=v1/schema/openAPIV3Schema/properties/spec/properties/upstreamresolvers/properties/upstreams/items/anyOf
+  value:
+  - properties:
+      type:
+        enum: [ "","SystemResolvConf" ] # empty string because client-side validation will not see the default value
+    not:
+      required: [ "address", "port" ]
+  - properties:
+      type:
+        enum: [ "Network" ]
+    required: [ "address" ]
+    optional: [ "port" ]
```

Comment:
Patch looks good. have you tried it in your tests?

Reply:
yep works ok